backend/src/databasefunctions.ts

```diff
@@ -0,0 +1,80 @@
+import { PoolConfig, ClientConfig } from 'pg';
+
+const { Pool } = require('pg');
+require('dotenv').config();
+
+const pgconfig: PoolConfig = {
+  user: process.env.DBUSER,
+  database: process.env.DATABASENAME,
+  password: process.env.DBPASSWORD,
+  host: process.env.DBHOST,
+  port: parseInt(process.env.DBPORT),
+};
+
+const pool = new Pool(pgconfig);
+
+pool.on('error', (err: Error, client: ClientConfig) => {
+  console.error(err, client);
+});
+export const createTable = async (sqlStatement: string) => {
+  if (
+    sqlStatement &&
+    !sqlStatement.toLowerCase().includes('create table')
+  ) {
+    throw new Error('wrong type of SQL statement');
+  }
+  let client;
+  try {
+    client = await pool.connect();
+    const res = await client.query(sqlStatement);
+    return res;
+  } finally {
+    try {
+      client.release();
+    } catch (err) {
+      console.error(err);
```

Comment:
How will this line (35) affect the rest of the application? Is the error still passed up to a centralized error handler or is it just lost at this point? If not, then I am assuming that no error message is passed to the front end, and users won't realize that their action was unsuccessful.

Same questions for the other console logs in the two other methods

Reply:
At this point every error both in the frontend and backend fail silently. 


I was assuming we were either going to have a logger (winston?) Or make a central error handler. Possibly both.


Since i havent talked to you about it yet i have not implemented either.